test/ably/restchannelpublish_test.py

```diff
@@ -111,10 +109,118 @@ def test_publish_various_datatypes_binary(self):
        self.assertEqual("This is a byte[] message payload",
                message_contents["publish4"],
                msg="Expect publish4 to be expected byte[]")
-       self.assertEqual({"test": "This is a JSONObject message payload"},
+       self.assertEqual('{"test": "This is a JSONObject message payload"}',
                json.loads(message_contents["publish5"]),
                msg="Expect publish5 to be expected JSONObject")
-       self.assertEqual(["This is a JSONArray message payload"],
+       self.assertEqual('["This is a JSONArray message payload"]',
                json.loads(message_contents["publish6"]),
                msg="Expect publish6 to be expected JSONObject")
 
+    def test_create_channel(self):
+        channelName = "name"
+        channel1 = TestRestChannelPublish.ably.channels.get(channelName)
+        existing = TestRestChannelPublish.ably.channels.get(channelName)
+        channel1.publish("msg", "msgData")
+        messages1 = channel1.history().current
+        self.assertIsNotNone(messages1, msg="Expected non-None messages from channel1")
+        messageContents1 = dict((m.name, m.data) for m in messages1)
+        messages2 = existing.history().current
```

Comment:
This API looks different to our other APIs, I would have expected `existing.history().items`